pkg/server/commands/write_test.go

```diff
@@ -238,13 +238,14 @@ func TestTransactionalWriteFailedError(t *testing.T) {
 	mockDatastore.EXPECT().
 		ReadAuthorizationModel(gomock.Any(), gomock.Any(), gomock.Any()).
 		Return(
-			testutils.MustTransformDSLToProtoWithID(`model
-	schema 1.1
+			testutils.MustTransformDSLToProtoWithID(`
+model
+schema 1.1
 type user
 
 type document
-  relations
-	define viewer: [user]`), nil)
+	relations
+		define viewer: [user]`), nil)
 
 	mockDatastore.EXPECT().
```

Comment:
I like the model definition to be aligned vertically with whatever comes before it

<img width="703" alt="image" src="https://github.com/openfga/openfga/assets/5374887/9491220f-2d5a-4dc1-8cc5-c75e93022131">

Reply:
FYI: accidentally closed PR. I will resubmit respecting previous lines indentation. ETA today/tomorrow